css/styles.css

```diff
@@ -13,6 +13,7 @@
     margin-bottom: 20px;
 }
 .header {
+    text-align: center;
```

Comment:
Please add a picture of what this does 🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠

Reply:
YES PLS